Maggie/error and email page

maggiehuang9 opened the pull request.

maggiehuang9 committed:
Modify parts list according to template.

maggiehuang9 committed:
Merge branch 'development' of https://github.com/ideahacks/ideahacks.la into maggie/inventory_page

maggiehuang9 committed:
change font and border size

maggiehuang9 committed:
change header to img

maggiehuang9 committed:
Changed color and style of email confirmation and error 404 page.

maggiehuang9 committed:
merge development

robertnperalta reviewed (changes requested):
Looks good! Just a few things:
* The error page looks great, but just scales down with screen size, so it gets tiny on mobile devices
* You need to swap the font over to Roboto Condensed for the confirm page
* I think you should change the style of the resend button on the confirm page to match the button on the landing page, just so everything matches

One other thing: I know that the parts page has already been merged and everything, but I couldn't help but notice that the icons overlap at around 1130px and that the layout isn't very mobile friendly. These issues do already exist on the current IDEAHacks page, so maybe you don't need to fix them, but I thought I'd mention it.

maggiehuang9 committed:
-changed confirm button to match style of landing page
-tested the error page on a mobile simulator and it did not seem to scale down, but if it still gets tiny I can look into it further

danielskathyd committed:
Merge conflicts

danielskathyd committed:
edit history pic

danielskathyd committed:
style fixes

danielskathyd committed:
run prettier

danielskathyd reviewed (approved):
lgtm
